include/mixer.h

```diff
@@ -177,20 +177,20 @@ class MixerChannel {
 	             const std::set<ChannelFeature>& features);
 	~MixerChannel();
 
-	bool HasFeature(ChannelFeature feature) const;
-	std::set<ChannelFeature> GetFeatures() const;
-	const std::string& GetName() const;
-	int GetSampleRate() const;
-	float GetFramesPerTick() const;
-	float GetFramesPerBlock() const;
-	double GetMillisPerFrame() const;
+	bool HasFeature(ChannelFeature feature);
+	std::set<ChannelFeature> GetFeatures();
```

Comment:
I thought you liked the use of `const`. I think `const` for member methods can be used by the compiler for optimization letting it know the object isn't mutable inside the function. I believe it translates the implicit `this` pointer into `const this`.